tests/integration/projects/general/service.py

```diff
@@ -116,6 +120,16 @@ def echo_with_delay(self, input_datas):
         time.sleep(data['b'] + data['a'] * len(input_datas))
         return input_datas
 
+    @bentoml.api(input=JsonInput())
+    def echo_json(self, input_data):
+        return input_data
+
+    if version.parse(BENTOML_VERSION) > version.parse("0.12.1+0"):
```

Comment:
What's the reason for this line here? All the Docker-based API server integration tests should be installing the current branch of BentoML right?  Is it meant for loading this class properly for the back-wards compatible tests?  If so, could you add an inline comment?